Drv/LinuxSerialDriver/LinuxSerialDriverComponentImplCommon.cpp

```diff
@@ -59,7 +59,7 @@ namespace Drv {
             }
         }
         this->m_readBuffMutex.unLock();
-        FW_ASSERT(found,Buffer.getbufferID(),Buffer.getmanagerID());
+        FW_ASSERT(found,Buffer.getContext());
```

Comment:
Why are we doing this check again?

Reply:
This check is to see if the buffer is found in the above for loop.  The buffer.getContext() is just the description provided as an arg, in the case it is not found, this is printed.

Reply:
I don't believe this is an issue, just an indication of the buffer identifier (context) that was not found.

Reply:
We would have to guarantee that 0 is never a valid value for the context.

Reply:
Why?  The getContext() is not part of the assert, it is for the message args.

Reply:
Duh! Brain short. Thought `getContext()` was the thing being asserted.